tensorflow_decision_forests/contrib/temporal_feature_processor/temporal_feature_processor/operators/window/base.py

```diff
@@ -0,0 +1,14 @@
+from pandas import Timedelta
+
+from temporal_feature_processor.sampling import Sampling
+
+from ..base import Operator
+
+
+class WindowOperator(Operator):
+    """Base class for window operators."""
+
+    def __init__(self, sampling: Sampling, window_length: Timedelta) -> None:
```

Comment:
It is not completely clear from this and the interfaces above how the input spec and input data are passed to either the object constructor or __call__method. Making a small unit test that demonstrates it would be very useful.

For example, I assume that the operator object will be accessible to the user (unless we only allow function constructors). For example, the user can write:

sales = t.ReadEvent(...)
sma_sales = t.SimpleMovingAverageOperator(data=sales, window_length=t.day(8))